client/landing/stepper/declarative-flow/helpers/index.ts

```diff
@@ -0,0 +1,13 @@
+import { STEPS } from '../internals/steps';
+
+export const shouldRedirectToSiteMigration = (
```

Comment:
It feels a bit weird to have this function that's somewhat flow-specific in a generic `helpers/index.ts` file. Might it make sense to have the function be in a standalone file? We can either import it directly from the file, or we can re-export it from a central `index.ts` file. Either way, we would only include functions that are actually used by the code, and we can tree-shake for bundling purposes too.

Given that this is a new file, it also feels like imports from here would only want one or two functions, not the whole file.

Reply:
I agree, I think the correct is to transform the site-migration-flow in a folder with an index file + this helper. It probably will help us to reduce the site-migration-flow file size.

Let me open another PR just with this change.